doc/source/whatsnew/v0.21.0.txt

```diff
@@ -138,3 +138,5 @@ Categorical
 
 Other
 ^^^^^
+
```

Comment:
move to numeric sub-section.

more like

Bug in ``.clip()`` when ``axis=1`` and a list-like is passed ; previously this raised ``ValueError``